code/src/components/Header.css

```diff
@@ -0,0 +1,17 @@
+.header-container{
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    text-align: center;
+    
+}
+
+.title{
+    font-family: 'Roboto';
+    font-weight: bolder;
+    font-size: 42px;
+    margin: 0;
+    color:whitesmoke;
+    
+}
```

Comment:
This file could be hidden or deleted since you don't seem to be using it in the final version?